apps/angular/src/core/services/anime.service.ts

```diff
@@ -0,0 +1,50 @@
+import { inject, Injectable } from '@angular/core';
+import { PaginationDto } from '@js-camp/core/dtos/pagination.dto';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Anime } from '@js-camp/core/models/anime';
+import { AnimeDto } from '@js-camp/core/dtos/anime.dto';
+import { PaginationMapper } from '@js-camp/core/mappers/pagination.mapper';
+import { AnimeMapper } from '@js-camp/core/mappers/anime.mapper';
+import { Pagination } from '@js-camp/core/models/pagination';
+import { AppUrlsConfig } from '@js-camp/angular/shared/app-url';
+import { AnimeFilterParams } from '@js-camp/core/models/anime-filter-params';
+
+import { AnimeFiltersParamsMapper } from '@js-camp/core/mappers/anime-filter-params.mapper';
+
+import { map, Observable } from 'rxjs';
+
+import { HttpParamsService } from './http-param.service';
+
+/** Anime services. */
+@Injectable({ providedIn: 'root' })
+export class AnimeService {
+	private readonly httpClient = inject(HttpClient);
+
+	private httpParamsService = inject(HttpParamsService);
+
+	private appUrlsConfig = inject(AppUrlsConfig);
+
+	private animeQueryMapper = inject(AnimeFiltersParamsMapper);
```

Comment:
Don't forget about `readonly` access modifier:
```suggestion
	private readonly httpParamsService = inject(HttpParamsService);

	private readonly appUrlsConfig = inject(AppUrlsConfig);

	private readonly animeQueryMapper = inject(AnimeFiltersParamsMapper);
```